cms/apps/media/models.py

```diff
@@ -35,20 +35,34 @@ class File(models.Model):
 
     """A static file."""
 
-    title = models.CharField(max_length=200,
-                             help_text="The title will be used as the default rollover text when this media is embedded in a web page.")
+    title = models.CharField(
+        max_length=200,
+        help_text='The title will be used as the default rollover text when this media is embedded in a web page.'
+    )
 
     labels = models.ManyToManyField(
         Label,
         blank=True,
-        help_text="Labels are used to help organise your media. They are not visible to users on your website.",
+        help_text='Labels are used to help organise your media. They are not visible to users on your website.',
     )
 
     file = models.FileField(
-        upload_to="uploads/files",
+        upload_to='uploads/files',
         max_length=250,
     )
 
+    width = models.CharField(
```

Comment:
I would have made these `PositiveIntegerField`s rather than `CharField`s, more appropriate for what they're being used for.